web/src/filter.js

```diff
@@ -868,7 +883,9 @@ export class Filter {
     update_email(user_id, new_email) {
         for (const term of this._operators) {
             switch (term.operator) {
+                case "dm-including":
                 case "group-pm-with":
+                case "dm":
                 case "pm-with":
                 case "sender":
                 case "from":
```

Comment:
I didn't delete either "pm-with" or "group-pm-with" here because the "from" operator was here, but I think all three could be removed after these changes because it's always called on a filter that's been set. And all three are updated in `Filter.canonicalize_term` when the filter is initialized.